Major copy changes to native.mdx

MindyCoW opened the pull request:
Rewrote a bunch of the page to simplify language & correct minor grammar errors

# Description

- Added more context regarding the pros and cons of wrapping native tokens vs. using the Eth-flow
- Fleshed out the steps for using the Eth-flow
- Simplified language where possible
- Corrected grammar errors

# Changes

<!-- List of detailed changes (how the change is accomplished) -->

- [ ] ...
- [ ] ...

<!--
## Related Issues

Fixes #
-->

MindyCoW committed:
Major copy changes to native.mdx

Rewrote a bunch of the page to simplify language & correct minor grammar errors

fedgiac commented:
Something I'd like to see in this page is that ETH can indeed be bought with CoW Swap without any issue. It's just selling that doesn't work well. Do you think you would be able to add it nicely or is it too much information?
Otherwise, I like the changes.

MindyCoW committed:
Added mfw's requested change to native.mdx

Included language emphasizing that buying native tokens on CoW Protocol is no problem

MindyCoW commented:
Good idea! Made the change. 

EDIT: Sorry Federico =( in the update title I attributed this suggestion to mfw =((

mfw78 commented:
No credit taken by me for @fedgiac's great work :smile:

mfw78 commented on docs/cow-protocol/tutorials/cow-swap/native.mdx:
```diff
@@ -6,38 +6,43 @@ import EthFlowTip from '../../../partials/_eth_flow.mdx'
 
-CoW Protocol is only capable of handling ERC20 tokens.
-This is because its intent-based architecture relies on solvers, or more precisely the vault relayer, to pull in the sell token on behalf of its traders.
+_Buying_ native tokens such as ETH on CoW Protocol is straightforward. However, CoW Protocol is only natively capable of _selling_ ERC20 tokens - not native tokens. 
```
```suggestion
_Buying_ native tokens such as ETH on CoW Protocol is straightforward. However, CoW Protocol is only capable of _selling_ ERC20 tokens - not native tokens. 
```

mfw78 commented on docs/cow-protocol/tutorials/cow-swap/native.mdx:
```diff
@@ -6,38 +6,43 @@ import EthFlowTip from '../../../partials/_eth_flow.mdx'
-In an attempt to smooth the user experience, CoW Protocol has introduced the [Eth-flow](/cow-protocol/reference/contracts/periphery/eth-flow) contract. This allows users to automate the [above process](#wrap-native-tokens), all in one **on-chain** transaction.
+In an attempt to smooth the user experience, CoW Protocol has introduced the [Eth-flow](/cow-protocol/reference/contracts/periphery/eth-flow) contract. This allows users to automate the [above process](#wrap-native-tokens) and wrap + swap in a single **on-chain** transaction. 
+
+Using the Eth-flow is faster than wrapping, then swapping. However, because an Eth-flow transactions is on-chain (unlike most CoW Swap swaps), the user must pay an up-front gas fee in the native token. This fee is not refunded if the order fails.
```
I'm not sure about the assertion of "faster" in this context?

mfw78 reviewed (approved):
Some minor points, will merge and then follow-up on discussion.
